WhenPushComesToShove/Assets/3-Scripts/MinigameLogic/VictoryRoomLogic.cs

```diff
@@ -22,20 +25,19 @@ public void Update()
 
     public override void Init()
     {
-        int highestScoreIndex = 0;
+        rankDisplay.ShowDisplay();
 
-        //Find Winning Team
-        for (int i = 1; i < GameState.playerScores.Length; i++)
-        {
-            if (GameState.playerScores[i] > GameState.playerScores[highestScoreIndex])
-            {
-                highestScoreIndex = i;
-            }
-        }
+        winningPlayer = GameState.players[rankDisplay.playerRankOrder[0]];
 
-        PlayerConfiguration winningPlayer = GameState.players[highestScoreIndex].GetComponentInChildren<PlayerInputHandler>().playerConfig;        
-        winnerText.text = GameState.playerNames[winningPlayer.PlayerIndex] + " player won!";
+        winningPlayer.GetChild(13).gameObject.SetActive(true);
 
         base.Init();
     }
+
+    public override void CleanUp()
+    {
+        winningPlayer.GetChild(13).gameObject.SetActive(false);
```

Comment:
I'm assuming this is for the crown. Is there a way to store this without getting GetChild?